version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "36.0.0"
+version in ThisBuild := "0.0.1"
```

Comment:
I'm cool with keeping the version at `0.0.1` until we are compiling this against an actual published `quasar` jar, at which point we should add a merge script to bump the version via tags like we do for `quasar`. Thoughts?

Reply:
I thought we agreed on this, actually :blush: sounds good to me.